addons/mass_mailing/data/mail_data.xml

```diff
@@ -17,4 +17,107 @@
             </body>
         </html>
     </template>
+
+    <!-- DIGEST TIPS -->
+
+    <record id="digest_tip_mass_mailing_0" model="digest.tip">
```

Comment:
Could be moved to digest_tip_data (data specific to a given model should be in a dedicated file).